php/ui/component/class-meta-box.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * Stand_Alone UI Component.
```

Comment:
<!-- trivial preference-based requests -->
**🥜 nitpick:** Wrong information about the package here and in the class :)